nilearn/datasets/atlas.py

```diff
@@ -1412,10 +1419,36 @@ def fetch_atlas_yeo_2011(data_dir=None, url=None, resume=True, verbose=1):
     %(resume)s
     %(verbose)s
 
+    n_networks : {7, 17, None}, default = None
```

Comment:
```suggestion
    n_regions : {7, 17, None}, default = None
```
I know "regions" and "networks" are sometimes used interchangeably in atlases but maybe it would be better to keep it consistent here?

Would require changes warnings and errors too

Reply:
for this one I used netwroks to try to match hat is used in this fetcher that mentions yeo's netwroks: https://nilearn.github.io/dev/modules/generated/nilearn.datasets.fetch_atlas_schaefer_2018.html#nilearn.datasets.fetch_atlas_schaefer_2018

also I believe that networks is more appropriate as several disconnected 'regions' do compose a network: see https://nilearn.github.io/dev/auto_examples/06_manipulating_images/plot_extract_regions_labels_image.html#sphx-glr-auto-examples-06-manipulating-images-plot-extract-regions-labels-image-py

what do you think?

Reply:
Agreed! Then I guess regions could changed to networks

Reply:
probably worth checking that the correct terminology is used across fetchers